Nye farge variabler

dagfrode opened the pull request:
# Fargeendringer
## ! Variabler som er fjernet
### Foreground
`--ffe-color-foreground-on-fill-default` erstattes med `--ffe-color-foreground-inverse`
`--ffe-color-foreground-on-fill-inverse` erstattes med `--ffe-color-foreground-default`

### Fill
`--ffe-color-fill-primary-hover` erstattes med `--ffe-color-fill-primary-default-hover` 
`--ffe-color-fill-primary-pressed` erstattes med `--ffe-color-fill-primary-default-pressed` 

### Border
Fjernet: 
`--ffe-color-border-secondary-*`
`--ffe-color-border-tertiary-*`

bruk 
`--ffe-color-border-primary-default`
`--ffe-color-border-primary-subtle`
`--ffe-color-border-primary-emphasis`
i stedet 

## ! Skjemaelementer - Gul border
Alle skjemaelementer har nå fått en gul focus-border. Komponentene blir nå lettere å se for brukere med nedsatt synsevne, og fokus-stilen fungerer på både hvit og blå bakgrunn (accent context). 

Alle skjemaelementer bruker også nå egne input-farger
`--ffe-color-component-form-input-fill-default`
`--ffe-color-component-form-input-foreground-default`
`--ffe-color-component-form-input-foreground-subtle`
`--ffe-color-component-form-input-border-active`

Checkbox (Ved keyboard focus)
<img width="207" height="81" alt="image" src="https://github.com/user-attachments/assets/5c8315c7-8712-4648-b49e-3c10a4952223" />

Input
<img width="457" height="107" alt="image" src="https://github.com/user-attachments/assets/08cd9675-e285-431a-88a0-b5721d924b91" />

InputTextLike
<img width="212" height="60" alt="image" src="https://github.com/user-attachments/assets/512fca5c-e2bf-4f16-9ff0-73d83f4e5084" />

Phonenumber
<img width="445" height="96" alt="image" src="https://github.com/user-attachments/assets/e9d69563-a1ee-4cc9-9bc5-5c4fd83f72ce" />

RadioBlock (nytt design) 
Eksisterende design 
<img width="750" height="350" alt="image" src="https://github.com/user-attachments/assets/2190c3a0-6d9f-4e07-8ef3-7d480413621a" />
Nytt design 
<img width="619" height="365" alt="image" src="https://github.com/user-attachments/assets/8b651b7f-e30a-4573-b18f-875327497c92" />

RadioButton (ved keyboard focus)
<img width="442" height="128" alt="image" src="https://github.com/user-attachments/assets/de6c2b5c-b556-4027-814c-a243a21ca5ab" />


RadioSwitch -(nytt design)
Eksisterende design
<img width="235" height="104" alt="image" src="https://github.com/user-attachments/assets/d5d1d9e3-2e98-41c5-bdd5-5a236d5b4431" />
Nytt design
<img width="229" height="86" alt="image" src="https://github.com/user-attachments/assets/2675343f-e77e-44f2-b373-092a726564ab" />

Textarea
<img width="454" height="119" alt="image" src="https://github.com/user-attachments/assets/153b7402-b575-4265-982d-3a92aeeeeb83" />

Tooltip (ny farge)
Eksisterende design
<img width="52" height="43" alt="image" src="https://github.com/user-attachments/assets/93ecc579-2049-4109-95e0-6898ad7bae0b" />
Nytt design
<img width="40" height="41" alt="image" src="https://github.com/user-attachments/assets/06fd0637-0f3d-4e31-abd1-a717641cee7e" />

## Label og button label - tykkere font
Eksisterende design 
<img width="139" height="43" alt="image" src="https://github.com/user-attachments/assets/a917c0ff-c1be-4319-9a21-709c884cd131" />
<img width="175" height="62" alt="image" src="https://github.com/user-attachments/assets/ae3f0efa-3c68-4a0b-842d-4162b83558c3" />

Nytt design 
<img width="144" height="37" alt="image" src="https://github.com/user-attachments/assets/fe05eb9d-7926-45a3-a54e-10cf63c22410" />
<img width="177" height="60" alt="image" src="https://github.com/user-attachments/assets/c1b8ef7e-da7e-440a-b87f-b8509a4dfb2e" />

# Nye fargevariabler
| Surface Neutral  | Farge |
| ------------- | ------------- |
| `--ffe-color-surface-neutral-default` | <img width="96" height="38" alt="image" src="https://github.com/user-attachments/assets/7ed837ba-0731-4eb6-b8b1-6280900eb0b4" />  |
| `--ffe-color-surface-neutral-default-hover`  | <img width="97" height="37" alt="image" src="https://github.com/user-attachments/assets/37f23095-34ec-4b3c-9021-d93b5eeb9baf" /> |
| `--ffe-color-surface-neutral-default-pressed` | <img width="97" height="36" alt="image" src="https://github.com/user-attachments/assets/673759c6-0899-43ea-83d8-f14baa7b3497" />  |


| Fill Neutral | Farge på hvit bakgrunn|
| ------------- | ------------- |
| `--ffe-color-fill-neutral-default` | <img width="96" height="37" alt="image" src="https://github.com/user-attachments/assets/5205c6f1-2bac-4b79-a071-6168303680fc" /> |
| `--ffe-color-fill-neutral-default-hover` | <img width="96" height="34" alt="image" src="https://github.com/user-attachments/assets/2c842667-81de-48fc-b81f-93de9ccd7418" /> |
| `--ffe-color-fill-neutral-default-pressed` | <img width="96" height="36" alt="image" src="https://github.com/user-attachments/assets/72c3cffb-75a9-4b15-93b8-e087e9409b61" /> | 
| `--ffe-color-fill-neutral-subtle` | <img width="95" height="32" alt="image" src="https://github.com/user-attachments/assets/7108457b-4bfe-40dd-9412-1e27319dcb56" /> | 
| `--ffe-color-fill-neutral-subtle-hover` | <img width="96" height="34" alt="image" src="https://github.com/user-attachments/assets/78beefac-8234-42a6-961e-bc6f345c6822" /> | 
| `--ffe-color-fill-neutral-subtle-pressed` | <img width="96" height="34" alt="image" src="https://github.com/user-attachments/assets/c096905f-40ac-438f-b193-38048358d473" /> | 
| `--ffe-color-fill-neutral-extra-subtle` | <img width="99" height="35" alt="image" src="https://github.com/user-attachments/assets/6b6ce5b0-93ff-42e4-948f-4ebb5ceb2cb1" /> |
| `--ffe-color-fill-neutral-extra-subtle-hover` | <img width="97" height="36" alt="image" src="https://github.com/user-attachments/assets/846b7ca2-5790-42cd-b3b5-2f45f7e7c13b" /> | 
| `--ffe-color-fill-neutral-extra-subtle-pressed` | <img width="96" height="32" alt="image" src="https://github.com/user-attachments/assets/2a56520c-ecf4-405a-98cf-4c71ca29592c" />| 


| Input | Farge |
| ------------- | ------------- |
| `--ffe-color-component-form-input-fill-default` | <img width="99" height="36" alt="image" src="https://github.com/user-attachments/assets/dada7f1c-5452-4775-b7c7-6c3280bc735d" /> |
| `--ffe-color-component-form-input-foreground-default` | <img width="97" height="38" alt="image" src="https://github.com/user-attachments/assets/017a9485-0e94-4947-ac4d-7870aac15aaf" /> |
| `--ffe-color-component-form-input-foreground-subtle` | <img width="99" height="35" alt="image" src="https://github.com/user-attachments/assets/87ba5afe-a3d9-4560-8f5c-f3f78716862c" /> |
| `--ffe-color-component-form-input-foreground-placeholder` | <img width="98" height="36" alt="image" src="https://github.com/user-attachments/assets/aede120d-4564-4e9e-ba7c-c5f87925de73" /> |
| `--ffe-color-component-form-input-foreground-disabled` | <img width="97" height="35" alt="image" src="https://github.com/user-attachments/assets/6875f63b-b286-4577-a60d-93e819f34740" /> |
| `--ffe-color-component-form-input-foreground-read-only` | <img width="96" height="34" alt="image" src="https://github.com/user-attachments/assets/4daa90b9-65a6-45eb-bb99-51735cb395f4" /> |
| `--ffe-color-component-form-input-foreground-read-only` | <img width="96" height="34" alt="image" src="https://github.com/user-attachments/assets/4daa90b9-65a6-45eb-bb99-51735cb395f4" /> |
| `--ffe-color-component-form-input-foreground-error` | <img width="96" height="35" alt="image" src="https://github.com/user-attachments/assets/224b969b-6478-41da-8763-ddb5a7232138" /> |
| `--ffe-color-component-form-input-border-active` |<img width="95" height="34" alt="image" src="https://github.com/user-attachments/assets/68b496d9-cb19-4033-96ab-e554846321b6" /> |

Obs: `--ffe-color-border-interactive-focus` får ny farge, og kan ikke brukes til active-states lenger. `component-input-border-active´(rett over) kan brukes til det.

dagfrode committed:
feat(ffe-core)!: Oppdaterer fargevariabler og farge scripts
BREAKING CHANGE: Fjerner noen fargevariabler og legger til nye fargevariabler. Se change log.
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
chore: fjern script for å synke tokens til figma

dagfrode committed:
feat(ffe-accordion)!: nye fargevariabler og fokusstil
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-buttons)!: nye fargevariabler og tykkere label
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-cards)!: nye fargevariabler
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
chore: forbedre stories for å gjøre lettere å oppdatere fargevariabler

dagfrode committed:
feat(ffe-chips)!: oppdartert fokusstate
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-datepicker)!: nye fargevariabler
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-form)!: tar i bruk nye fargevariabler og ny fokusstate
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-form-react)!: oppdatert design på radioblock
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-messages-react)!: endrer til default mode i accent context
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-header)!: nye fargevariabler
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-tabs)!: nye fargevariabler
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-searchable-dropdown-react)!: nye fargevariaber, fokusstate og aria-invalid state
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

tuva-odegard committed:
feat(ffe-dropdown-react)!: nye fargevariabler, fokusstate og aria-invalid state
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

tuva-odegard commented on packages/ffe-buttons/less/base-button.less:
```diff
@@ -330,8 +330,8 @@
             border-radius: 50%;
-            height: 45px;
-            width: 45px;
+            height: 48px;
```
Er det med vilje at denne skal være større? Ble den ikke veldig stor?

dagfrode replied:
Det er med vilje, ble gjort med Ragnhild. Den hadde mye padding tidligere så det så rart ut med focus. gjorde knappen litt større og fjernet paddingen.

dagfrode replied:
oppdatert til venstre. dagens til høyre
<img width="1604" height="536" alt="image" src="https://github.com/user-attachments/assets/8b2dae74-25e1-4a58-bcb8-87619b5d02d7" />

tuva-odegard replied:
ahh skjønner! Den er 3 px høyere nå da.. Den er i utgangspunktet ganske høy, kunne man ikke bare fjernet paddingen og latt være å øke fra 45 til 48? 🤔

tuva-odegard commented on packages/ffe-form-react/src/RadioSwitch.tsx:
```diff
@@ -46,9 +46,14 @@ export const RadioSwitch: React.FC<RadioSwitchProps> = ({
+                className={classNames(
+                    'ffe-radio-switch',
+                    'ffe-default-mode',
+                    className,
```
Dette burde kanskje være i en commit relatert til ffe-form-react?

dagfrode replied:
Oj, er den ikke det? :see_no_evil: Ja!

tuva-odegard committed:
chore: oppdaterer changelog

dagfrode committed:
feat(ffe-chart-donut-react)!: nye fargevariabler

BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

dagfrode committed:
feat(ffe-tables)!: nye fargevariabler

BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

tuva-odegard committed:
feat(ffe-spinner)!: endrer hovedfargen på spinneren.
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs

tuva-odegard committed:
fix(ffe-pagination)!: Endrer selected-fargene på knappene så det blir
riktig i dark mode
BREAKING CHANGE: Breaking pga. oppdateringer i ffe-core som fjerner noen fargevariabler og legger til nye fargevariabler. Se change log
https://sparebank1.github.io/designsystem/?path=/docs/introduksjon-changelog--docs
